SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Security Policy
+
+## Security Release Process
+
+Clarity is an open source design system that brings together UX guidelines, design resources, and coding implementations with Web Components. This repository includes everything you need to build, customize, test, and deploy Clarity. For complete documentation, visit the Clarity website.
+
+The community has adopted this security disclosure and response policy to ensure we responsibly handle critical issues.
```

Comment:
Add this section above "Security release process" because it's more like informational about the project.

Reply:
corrected in commit 47fba4e